server.js

```diff
@@ -1,8 +1,13 @@
 import express from "express";
 import cors from "cors";
+import dotenv from "dotenv";
 import mongoose from "mongoose";
+import listEndpoints from "express-list-endpoints";
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+dotenv.config()
+
+// const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+const mongoUrl = process.env.MONGO_URL || `mongodb+srv://${process.env.MONGO_USER}:${process.env.MONGO_PASS}@cluster0.eilze2r.mongodb.net/project-happy-thoughts-api?retryWrites=true&w=majority`;
```

Comment:
you could add the url for the database as MONGO_URL in the .env file instead, like this:

`MONGO_URL = `mongodb+srv://**USERNAME**:**PASSWORD**@cluster0.eilze2r.mongodb.net/project-happy-thoughts-api?retryWrites=true&w=majority``